crates/sui-graphql-e2e-tests/tests/stable/consistency/multi_get_objects.move

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --protocol-version 51 --addresses Test=0x0 --accounts A B --simulator --objects-snapshot-min-checkpoint-lag 4
+
+
+// cp | object_id | owner
+// ----------------------
+// 1  | obj_2_0   | A
+// 1  | obj_3_0   | A
+// 2  | obj_6_0   | A
+// 2  | obj_7_0   | A
+// All owned by B after checkpoint 3.
+
+//# publish
+module Test::M1 {
+    public struct Object has key, store {
+        id: UID,
+        value: u64,
+    }
+
+    public entry fun create(value: u64, recipient: address, ctx: &mut TxContext) {
+        transfer::public_transfer(
+            Object { id: object::new(ctx), value },
+            recipient
+        )
+    }
+}
+
+//# run Test::M1::create --args 0 @A
+
+//# run Test::M1::create --args 1 @A
+
+//# create-checkpoint
+
+//# run-graphql --cursors bcs(@{obj_2_0},@{highest_checkpoint}) bcs(@{obj_3_0},@{highest_checkpoint})
```

Comment:
What is this part trying to test? It doesn't seem related to multi-getting objects.